app/frontend/src/components/Avatar/Avatar.tsx

```diff
@@ -14,16 +14,27 @@ const useStyles = makeStyles({
     position: "absolute",
     top: 0,
     width: "100%",
+    maxWidth: "288px",
```

Comment:
How did you pick 288? Maybe leave a comment above it

Reply:
It is 12 times our 24px grid. I could leave it as `theme.spacing(36) `

Reply:
Ok! Actually maybe make it `18rem` - that way it will respond to user don't size preferences, and won't break if we want to change our spacing later